Scripts/AwsEC2GetPublicSGRules/AwsEC2GetPublicSGRules.yml

```
@@ -0,0 +1,92 @@
+args:
+- default: false
+  description: Security Group ID (sg-xxxxxxxxx)
+  isArray: false
+  name: groupId
+  required: true
+  secret: false
+- default: false
+  description: JSON string of the ipPermissions
+  isArray: true
+  name: ipPermissions
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: false
+  description: Protocol to check. TCP/UDP/All(-1)
+  isArray: false
+  name: protocol
+  predefined:
+  - tcp
+  - udp
+  - '-1'
+  required: true
+  secret: false
+- default: false
+  description: Lower bound port range to be checked. If fromPort and toPort are not
+    specified, all port will be included.
+  isArray: false
+  name: fromPort
+  required: false
+  secret: false
+- default: false
+  description: Upper bound port range to be checked. If fromPort and toPort are not
+    specified, all port will be included.
+  isArray: false
+  name: toPort
+  required: false
+  secret: false
+- default: false
+  description: Security group region
+  isArray: false
+  name: region
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: true
+  defaultValue: 'no'
+  description: Include IPv6 in the result. By default, IPv6 is not included
+  isArray: false
+  name: includeIPv6
+  predefined:
+  - 'yes'
+  - 'no'
+  required: false
+  secret: false
+comment: Find Security Group rules which allows ::/0 (IPv4) or 0.0.0.0/0.
+commonfields:
+  id: 0d62193f-6d73-477d-8020-2261f7d885c3
```

please change the id the be the name of the script
```suggestion
  id: AwsEC2GetPublicSGRules
```